designcompose/src/main/java/com/android/designcompose/Jni.kt

```diff
@@ -35,32 +36,92 @@ internal class TextSize(
     var height: Float = 0F,
 )
 
+internal interface UniInterface {
```

Comment:
What is `UniInterface`? Why do you need an interface if `TracedJni` is the only type exported from this file?

Reply:
Wow, I replaced the typo with another typo.

I was hesitant to unilaterally add the tracing code around the JNI interface and thought it might be best to leave the traced and untraced versions. I'm not really sure if I need the interface actually, I was kind of figuring it out. I suppose I could just have TracedJNI be a subclass. Or I could just move the tracing versions of the functions into the single JNI class.

Reply:
I'd just export one class with the methods you want, and have tracing on the ones that we know are expensive. You can subclass and override, but my expectation would be that if a method name starts `jni` then it's native code, and I'd be confused by a stacktrace with Java code in a `jni`-prefixed function that I know doesn't call `Call*Method`. (I would think "ugh! the stack has been corrupted!").

Reply:
How's this?